src/components/Form.js

```diff
@@ -16,7 +20,16 @@ function Form() {
     e.preventDefault();
 
     // submit data
-    // if (!title.trim() || !author.trim()) return;
+    if (!title.trim() || !author.trim()) return;
+
+    const bookData = {
+      item_id: uuidv4(),
+      title,
+      author,
+    };
+
+    // Adding book to state
+    dispatch(booksActions.addBook(bookData));
```

Comment:
- [ OPTIONAL ] Great job 👏🏻👏🏻 but I highly recommend after adding a new book to make the inputs empty so not let the user earase what he/she wrote and write again so that will imporve from user experience so kindly update it 👏🏻💯🤩✨

Reply:
Thank you @Reem-lab for the feedback. I will add the recommended change.